modular_skyrat/modules/goofsec/code/sec_clothing_overrides.dm

```diff
@@ -470,6 +470,44 @@
 	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/hands.dmi'
 	icon_state = "tackle_blue"
 
+//BUBBER ADDITION START
+	uses_advanced_reskins = TRUE
+	unique_reskin = list(
+		"Black Variant" = list(
+			RESKIN_ICON_STATE = "combat",
+			RESKIN_WORN_ICON_STATE = "combat"
+		),
+		"Blue Variant" = list(
+			RESKIN_ICON_STATE = "tackle_blue",
+			RESKIN_WORN_ICON_STATE = "tackle_blue"
+		),
+		"Red Variant" = list(
+			RESKIN_ICON_STATE = "gorilla",
+			RESKIN_WORN_ICON_STATE = "gorilla"
+		),
+	)
+
+/obj/item/clothing/gloves/tackler/combat
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/gloves.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/hands.dmi'
+	icon_state = "gorilla"
+	uses_advanced_reskins = TRUE
+	unique_reskin = list(
+		"Black Variant" = list(
+			RESKIN_ICON_STATE = "combat",
+			RESKIN_WORN_ICON_STATE = "combat"
+		),
+		"Blue Variant" = list(
+			RESKIN_ICON_STATE = "tackle_blue",
+			RESKIN_WORN_ICON_STATE = "tackle_blue"
+		),
+		"Red Variant" = list(
```

Comment:
you pretty much added the TG version version of the sprite to the skyrat file right? why not just use RESKIN_ICON to specify the icon it uses